README.md

```diff
@@ -124,6 +124,7 @@ We recommend you either work in a virtual environment, or on a dual booted syste
 2. [Dual Booting](https://www.tecmint.com/install-ubuntu-alongside-with-windows-dual-boot/amp/), [option2](https://askubuntu.com/questions/1031993/how-to-install-ubuntu-18-04-alongside-windows-10), [video guide](https://www.youtube.com/watch?v=qNeJvujdB-0&fbclid=IwAR0APhs89jlNR_ENKbSwrp6TI6P-wxlx-a0My9XBvPNAfwtADZaAXqcKtP4)
 3. [Setting up a Linux virtual env](https://itsfoss.com/install-linux-in-virtualbox/)
 
+
```

Comment:
Were you supposed to add this space in the readme?

Reply:
I did not add any space in the readme

Reply:
[Here](https://github.com/publiclab/plots2/pull/11516/files/c5231ee1bd2bd54d3f3ff84b43202247829d1eec#diff-b335630551682c19a781afebcf4d07bf978fb1f8ac04c6bf87428ed5106870f5) at line 127 it shows you added a line.